### PR TITLE
DHUB-73 Replace Oracle Java 8 in the interactive-notebooks

### PR DESCRIPTION
**Pull request by mcoberly2**

### CODE REVIEW

**Comment by arrowplum** [comment]
Where did we get that jdk?

**Review by rahul-aerospike — APPROVED** [review]
Looks alright to me.

**Comment by mcoberly2** [comment]
> Where did we get that jdk?

@rahul-aerospike and I both did a Web search and this was our result.

Do you have a recommendation?
We are more than happy to go with your recommendation.

File: Dockerfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ RUN chown -R ${NB_UID} ${HOME}
 
 # spark notebook
 RUN mkdir /opt/spark-nb; cd /opt/spark-nb\
-  && wget -qO- "https://javadl.oracle.com/webapps/download/AutoDL?BundleId=245467_4d5417147a92418ea8b615e228bb6935" | tar -xvz \
+  && wget -qO- "https://builds.openlogic.com/downloadJDK/openlogic-openjdk/8u362-b09/openlogic-openjdk-8u362-b09-linux-x64.tar.gz" | tar -xvz \
```
Comment (mcoberly2) [inline Dockerfile:26]:
```suggestion
  && wget -qO- "https://cdn.azul.com/zulu/bin/zulu8.70.0.23-ca-jdk8.0.372-linux_x64.tar.gz" | tar -xvz \
```

Comment (mcoberly2) [reply Dockerfile:26]: This was added per recommendation from Joe Martin.

**Comment by mcoberly2** [comment]
Closing per further discussion with @rahul-aerospike as the JDK8 was being overwritten to JDK11 via this line: https://github.com/aerospike-examples/interactive-notebooks/blame/b7ce9196e0d70e8c14c1831387523a6811e9d991/Dockerfile#L45

JDK was only required for Aerospike Connect for Spark running with Apache Spark 3.0. 

Since Aerospike Connect for Spark 3.1.1 we have been running with Apache Spark 3.1

**Pull request closed without merging**